engine/execution/state/delta/view.go

```diff
@@ -68,9 +68,7 @@ func (v *View) Interactions() *SpockSnapshot {
 		reads[i] = id
 	}
 
-	spockSecHashSum := v.spockSecretHasher.SumHash()
-	var spockSecret = make([]byte, len(spockSecHashSum))
-	copy(spockSecret, spockSecHashSum)
```

Comment:
I don't remember the reason we were doing this in the first place, any place this might be reused?

Reply:
I remember I commented on this but I couldn't find the old PR. It got merged and we said let's remove the extra copy later.